taxonomy-editor-frontend/src/pages/startproject/index.jsx

```diff
@@ -0,0 +1,100 @@
+import { Typography, Box, Grid, TextField, Stack, Autocomplete, Snackbar, Alert } from "@mui/material";
+import LoadingButton from '@mui/lab/LoadingButton';
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { TAXONOMY_NAMES } from "../../constants";
+import { createBaseURL } from "../editentry/createURL";
+
+const StartProject = () => {
+    const [branchName, setBranchName] = useState("")
+    const [taxonomyName, setTaxonomyName] = useState("additives")
+    const [description, setDescription] = useState("");
+    const [loading, setLoading] = useState(false); 
+    const [open, setOpen] = useState(false);
```

Comment:
what about just open the thing when there's an error message, and when closing remove set error message to null? that way we can remove 1 state and make the logic cleaner?

Reply:
Yep, removed the `open` state and used a single state